data_steward/cdr_cleaner/cleaning_rules/drop_participants_without_ppi_or_ehr.py

```diff
@@ -0,0 +1,92 @@
+"""
+Run the drop_participants_without_ppi_or_ehr validation clean rule.
```

Comment:
@mark-velez please review the approach, let me know if there is any better way to proceed here you can think of.